### PR TITLE
 skeleton of music app cluster

### PR DESCRIPTION
**Pull request by bhassan6621**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ class App extends Component {
           <h2>Welcome to React</h2>
           {this.state.text}
         </div>
+        <MusicApps />
```
Comment (ajwang0) [inline client/src/App.js:26]: Move MusicApps below Top Charts. I think TopCharts is one of the first things in the mock

File: client/src/Clusters/MusicApps/MockData.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            appId: "03",
+            wearOsImage: smartWatch,
+            title: "NavMusic: Wear OS Offline...",
+            iconUrl: navMusicIcon,
+            rating: 4.6,
+            rated: "E - Everyone",
+            appDescription: "Play offline on your wear OS device with a clear, simple interface"
+        },
+        {           
+            appId: "04",
```
Comment (ajwang0) [inline client/src/Clusters/MusicApps/MockData.js]: Let's make this consistent and just call this "id" everywhere

File: client/src/Clusters/MusicApps/SingleCardContainer.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    render() {
+        const {bigImage, title, iconUrl, rating, rated, appDescription} = this.props;
+        return (
+          <>
+            <img class="musicImage" alt="largeImage" src={bigImage} />
+            <div class="appCard">
+                <img class="img-fluid" alt="appIcon" src={iconUrl} />
+                <div class="app-details">
+                     <h3 class="app-title">{title}</h3>
+                     <p class="grey-text">
```
Comment (ajwang0) [inline client/src/Clusters/MusicApps/SingleCardContainer.js]: I would actually change this is a div since it's technically not a paragraph

**Review by ajwang0 — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (bhassan6621) [reply client/src/App.js:26]: top charts is actually one of the last things in the mock, music apps are above it

**Review by ajwang0 — APPROVED** [review]

**Pull request merged**